tests/common.tests.ts

```diff
@@ -134,192 +134,79 @@ export class CommonTests {
+        const results = <SchemaRecord[]>response.results;
+
+        assert.ok(results && results.length, `Page ${page + 1}: Have results returned`);
+        assert.equal(
+          providerConfig[testConfig.batchSizeLimitAttribute],
+          results.length,
+          `Page ${page + 1}: Have correct number of results returned`
         );
-      }
 
-      CommonTests.checkItem(results[0], handler, provider)
-
-      assert.equal(
-        SyncHandlerStatus.SYNCING,
-        response.position.status,
-        "Sync is active"
-      );
-      assert.ok(response.position.thisRef, "Have a defined processing range");
-
-      const currentRangeParts = response.position.thisRef!.split(':')
-      assert.ok(currentRangeParts.length == 2, "Have correct number of parts for the processing range");
-      assert.ok(currentRangeParts[0] == results[0]._id.replace(`${idPrefix}-`, ''), "Have correct break ID");
-      assert.ok(currentRangeParts[1].length, "Have an end range");
-
-      // 2. Backfill items are processed
-      const syncPosition2 = response.position
-      const response2 = await handler._sync(api, syncPosition2);
-      const results2 = <SchemaRecord[]>response2.results;
-      
-      // console.log(response2.position)
-      // console.log(CommonTests.outputItems(results2, testConfig.timeOrderAttribute))
-
-      assert.ok(
-        results2 && results2.length,
-        "Have backfill results returned"
-      );
-      assert.ok(
-        results2 &&
-          results2.length == providerConfig[testConfig.batchSizeLimitAttribute],
-        "Have correct number of results returned in second page"
-      );
-
-      if (testConfig.timeOrderAttribute) {
-        assert.ok(
-          results2[0][testConfig.timeOrderAttribute] >
-            results2[1][testConfig.timeOrderAttribute],
-          "Results are most recent first"
-        );
-        assert.ok(
-          results2[0][testConfig.timeOrderAttribute] <
-            results[2][testConfig.timeOrderAttribute],
-          "First item on second page of results have earlier timestamp than last item on first page"
-        );
-      }
+        if (testConfig.timeOrderAttribute) {
+          assert.ok(
+            results[0][testConfig.timeOrderAttribute] >
+              results[1][testConfig.timeOrderAttribute],
+            `Page ${page + 1}: Results are most recent first`
+          );
+        }
 
-      assert.equal(
-        response2.position.status,
-        SyncHandlerStatus.SYNCING,
-        "Sync is active"
-      );
-
-      assert.ok(response2.position.thisRef, "Have a defined processing range");
-
-      const currentRangeParts2 = response2.position.thisRef!.split(':')
-      assert.ok(currentRangeParts2.length == 2, "Have correct number of parts for the processing range");
-      assert.ok(currentRangeParts2[0] == results[0]._id.replace(`${idPrefix}-`, ''), "Have correct break ID matching the very first result");
```

Comment:
This test has been lost. It checks the second page of results doesn't change the break Id, which was set using the first page of reuslts.